packages/sdk/src/model/ModelClient.ts

```diff
@@ -389,6 +392,35 @@ export class ModelClient extends APIResource {
     }
   }
 
+  /* -------------------------------------------------------------------------
+   * List Model Runs
+   * -----------------------------------------------------------------------*/
+
+  async listModelRunsQuery({
+    modelName,
+    fullView,
+    pageSize,
+    page,
+    orderBy,
+    filter,
+  }: ListModelRunsRequest) {
+    try {
+      const queryString = getQueryString({
+        baseURL: `${modelName}/runs${fullView ? "?view=VIEW_FULL" : ""}`,
+        pageSize,
+        page,
+        filter,
+        orderBy,
+      });
```

Comment:
@orangecoloured As the SDK need to fully obey what API defined its parameter. We didn't have fullView as the official exposed parameters. So this need to be replaced with exposing `view` instead and we have that param in the getQueryString too